### PR TITLE
sig-cluster-lifecycle: update list of chairs

### PR DESCRIPTION
**Pull request by neolit123**
/assign @justinsb @timothysc

### CODE REVIEW

File: sig-cluster-lifecycle/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The Cluster Lifecycle SIG examines how we should change Kubernetes to make it ea
 ### Chairs
 The Chairs of the SIG run operations and processes governing the SIG.
 
-* Robert Bailey (**[@roberthbailey](https://github.com/roberthbailey)**), Google
+* Justin Santa Barbara (**[@justinsb](https://github.com/justinsb)**), Google
```
Comment (nikhita) [inline sig-cluster-lifecycle/README.md]: Consider adding Robert as an emeritus chair? :)

https://github.com/kubernetes/community/blob/91e739ad97d2418b04c0574dab5b364c1be66fe4/sigs.yaml#L325

Comment (neolit123) [reply sig-cluster-lifecycle/README.md]: looks like this file is generated...
> Consider adding Robert as an emeritus chair? :)

SGTM, will do.

**Comment by neolit123** [comment]
/assign @nikhita 
updated PTAL.

**Comment by neolit123** [comment]
thanks
/hold cancel

**Pull request merged**